MPChartLib/src/com/github/mikephil/charting/renderer/PieChartRenderer.java

```diff
@@ -919,10 +919,15 @@ protected void drawRoundedSlices(Canvas c) {
      * Releases the drawing bitmap. This should be called when {@link LineChart#onDetachedFromWindow()}.
      */
     public void releaseBitmap() {
+        if (mBitmapCanvas != null) {
+            mBitmapCanvas.setBitmap(null);
+            mBitmapCanvas = null;
+        }
         if (mDrawBitmap != null) {
             mDrawBitmap.get().recycle();
             mDrawBitmap.clear();
             mDrawBitmap = null;
         }
+        System.gc();
```

Comment:
This does not cause performance issues? http://stackoverflow.com/questions/2414105/why-is-it-bad-practice-to-call-system-gc